app/src/main/java/org/eyeseetea/malariacare/layout/adapters/survey/DynamicTabAdapter.java

```diff
@@ -716,6 +716,13 @@ public View getView(int position, View convertView, ViewGroup parent) {
                     showOrHideChildren(screenQuestion);
                     break;
             }
+            if(screenQuestion.getCompulsory()!=null && screenQuestion.getCompulsory() == 0) {
+                ImageView rowCompulsoryView = ((ImageView) tableRow.findViewById(R.id.row_header_compulsory));
```

Comment:
@gnarbaiz please note this line breaks Laos & Cambodia variants. You cannot call from main an id that's only present on one variant. You need to move that part to a function only present on myanmar variant, so only myanmar will see the "R.id.row_header_compulsory" search.